stubs/docutils/docutils/nodes.pyi

```diff
@@ -107,6 +107,9 @@ class Element(Node):
     def astext(self) -> str: ...
     def __getattr__(self, __name: str) -> Incomplete: ...
 
+class TextElement(Element):
+    def __init__(self, rawsource: str = "", text: str = "", *children, **attributes): ...
```

Comment:
The `children` argument is just passed to `Element.__init__`, and we have the `children` argument of `Element.__init__` annotated with Node further up in this file

```suggestion
    def __init__(self, rawsource: str = "", text: str = "", *children: Node, **attributes) -> None: ...
```